docs/stake-pool-guide/cardano-relay-configuration.md

```diff
@@ -0,0 +1,216 @@
+---
+id: cardano-relay-configuration
+title: Cardano Relay Configuration (Startup scripts, Systemd)
+sidebar_label: Cardano Relay Configuration
+description: Cardano Relay Configuration (Startup scripts, Systemd)
+image: ../img/og-developer-portal.png
+---
+In the last section, we went through the process of downloading, compiling, and installing cardano-node and cardano-cli into your Linux-based operating system. Now in this section we will walk through the process of configuring the installed node as a relay node.
+
+## Configuration Files
+
+The `cardano-node` application requires the following configuration files to run as of writing this article.
+
+- **Main Config**: It contains general node settings such as **logging** and **versioning**. It also points to the **Byron Genesis** and the **Shelly Genesis** file.
+- **Byron Genesis**: It contains the initial protocol parameters and instructs the `cardano-node` on how to bootstrap the **Byron Era** of the **Cardano** blockchain.
+- **Shelly Genesis**: It contains the initial protocol parameters and instructs the `cardano-node` on how to bootstrap the **Shelly Era** of the **Cardano** blockchain.
+- **Alonzo Genesis**: It contains the initial protocol parameters and instructs the `cardano-node` on how to bootstrap the **Alonzo Era** of the **Cardano** blockchain.
+- **Topology**: It contains the list of network peers (**`IP Address` and `Port` of other nodes running the blockchain network**) that your node will connect to.
+
+:::important
+Currently, the `cardano-node` topology is manually set by the community of network operators in the **Cardano** blockchain. But an automated p2p (peer-to-peer) system is in the works. For more information visit, [Boosting network decentralization with P2P](https://iohk.io/en/blog/posts/2021/04/06/boosting-network-decentralization-with-p2p/).
```

Comment:
There is a more recent blog article from IOG about the P2P capabilities & development status:
```suggestion
Currently, the `cardano-node` topology is manually set by the community of network operators in the **Cardano** blockchain. But an automated p2p (peer-to-peer) system is in the works. For more information visit, [Introducing our new peer-to-peer (P2P) testnet](https://iohk.io/en/blog/posts/2021/12/08/introducing-our-new-peer-to-peer-p2p-testnet/).
```